Documentation/environment-variables.rst

```diff
@@ -21,47 +21,24 @@ In addition to the variables in SeisSol, the
 `ASYNC <https://github.com/TUM-I5/ASYNC>`__ library provides some tuning
 variables listed in the `wiki <https://github.com/TUM-I5/ASYNC/wiki>`__.
 
-Typical example
-^^^^^^^^^^^^^^^
-
-We want to run a 130 nodes job. We dedicate 2 nodes for writing the
-asynchronous outputs, the remaining 128 nodes for computing. We then use
-a mesh partitioned in 128 regions. In our batch file, we write the
-following environment variables:
-
-
-.. code:: bash
-
-   export ASYNC_MODE=MPI
-   export ASYNC_MPI_COPY=1
-   export ASYNC_GROUP_SIZE=64
-   export XDMFWRITER_ALIGNMENT=8388608
-   export XDMFWRITER_BLOCK_SIZE=8388608
-
-Note that in the current implementation, at least 2 output nodes have to
-be used.
-
-
 Checkpointing
 ~~~~~~~~~~~~~
 
 Some environment variables related to checkpointing are described in the :ref:`Checkpointing section <Checkpointing>`.
 
+.. _optimal_environment_variables_on_supermuc_ng:
 
-Optimal environment variables on SuperMuc
------------------------------------------
-
-NG
-~~
+Optimal environment variables on SuperMuc-NG
+--------------------------------------------
 
-On NG, we recommend using SeisSol with asyn output in thread mode.
-That is SeisSol should be compiled with commThread='yes', and then run with the environment variables proposed below.
+On SuperMuc-NG, we recommend using SeisSol with async output in thread mode.
+That is SeisSol should be compiled with :code:`-DCOMMTHREAD=ON`, and then run with the environment variables proposed below.
 Also, we recommend using hyperthreading capabilities (that is using 96 CPUs instead of 48. 2 threads out of 96 are used as communication threads).
 Here are some proposed environment variables, to be added prior to invoking SeisSol in your batch file:
 
 .. code:: bash
 
-   #SBATCH --nodes=
+   #SBATCH --nodes=<number of nodes>
    #SBATCH --ntasks-per-node=1 
    #SBATCH --cpus-per-task=96
```

Comment:
```#SBATCH --cpus-per-task=96``` could (should?) be removed.
You could also add a link to https://seissol.readthedocs.io/en/latest/supermuc.html#running-seissol